src/Uno.UI.Runtime.Skia.Tizen/Tizen/UnoCanvas.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ElmSharp;
+using SkiaSharp;
+using SkiaSharp.Views.Tizen;
+using WUX = Windows.UI.Xaml;
+
+namespace Uno.UI.Runtime.Skia
+{
+	public class UnoCanvas : SKCanvasView
+	{
+		private bool _sizeInitialized = false;
+
+		public UnoCanvas(EvasObject parent) : base(parent)
+		{
+			PaintSurface += UnoCanvas_PaintSurface;
+			Resized += UnoCanvas_Resized;
+
+			WUX.Window.InvalidateRender
+				+= () =>
+				{
+					Invalidate();
+				};
+		}
+
+		private void UnoCanvas_Resized(object sender, EventArgs e)
+		{
+			var c = (SKCanvasView)sender;
+
+			var geometry = c.Geometry;
+
+			// control is not yet fully initialized
+			if (geometry.Width <= 0 || geometry.Height <= 0)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5993935)